cl/search/templates/cotton/corpus_search/scope.html

```diff
@@ -3,9 +3,54 @@
 {% require_script "js/alpine/composables/focus.js" %}
 {% require_script "js/alpine/plugins/focus@3.14.8" defer=True %}
 
-<c-vars class="" menu_class="" item_class=""></c-vars>
+<c-vars class="" menu_class="" item_class="" variant="menu"></c-vars>
 
 <div class="relative">
+
+{% if variant == "tabs" %}
+
+  <div
+    data-title="{{ title }}"
+    role="tablist"
+    aria-label="Select the scope of your search"
+    class="w-full gap-1 items-stretch {{ class }}"
+    x-data="focus"
+    x-bind:aria-activedescendant="selectedScope"
+    x-on:keydown.left="focusPrevious"
+    x-on:keydown.right="focusNext"
+    x-on:keydown.home.prevent="focusFirst"
+    x-on:keydown.end.prevent="focusLast"
+  >
+    <template x-for="scope in searchScopes">
+      <button
+        x-on:click="selectScope"
+        x-bind:data-scope="scope.label"
+        x-bind:id="scope.label"
+        x-bind:aria-selected="isActiveScope"
+        type="button"
+        role="tab"
+        class="bg-white px-8 min-w-41 h-[58px] text-sm font-normal rounded-t-2xl text-greyscale-600 flex justify-center items-center"
+      >
+        <span class="flex flex-col items-start">
+          <span
+            x-bind:data-scope="scope.label"
+            x-bind:class="scopeTabClass"
+            x-text="scope.label"
+            class=""
+          ></span>
+          <span
+            x-show="isActiveScope"
+            x-bind:data-scope="scope.label"
+            x-text="scope.shortDescription"
+            class="text-xs text-greyscale-500"
+          ></span>
+        </span>
+      </button>
```

Comment:
I noticed that our tabs currently use the same color for both the **selected** and **unselected** states. At first, I thought it might just be my impression, but I double-checked the Figma mockup and confirmed it that the inactive tab should use a slightly warmer tone. Here's a video for reference:



https://github.com/user-attachments/assets/a0f612c0-323e-4cba-a6a9-8591e2ae9ec6